dbt_subprojects/daily_spellbook/models/_sector/dex/pools/optimism/uniswap/_sources.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+ 
+sources:
+  - name: uniswap_v3_optimism
+    tables:
+      - name: UniswapV3Factory_evt_PoolCreated
+  - name: tokens_optimism
+    tables:
+      - name: balances_daily
```

Comment:
please move this source to:
https://github.com/duneanalytics/spellbook/blob/main/sources/_subprojects_outputs/tokens/balances.yml